Source/Core/Core/IOS/WFS/WFSI.cpp

```diff
@@ -216,14 +216,23 @@ IPCCommandResult WFSI::IOCtl(const IOCtlRequest& request)
     break;
 
   case IOCTL_WFSI_INIT:
+  {
     INFO_LOG(IOS, "IOCTL_WFSI_INIT");
     if (GetIOS()->GetES()->GetTitleId(&m_title_id) < 0)
     {
       ERROR_LOG(IOS, "IOCTL_WFSI_INIT: Could not get title id.");
       return_error_code = IPC_EINVAL;
       break;
     }
+    m_title_id_str = StringFromFormat(
+        "%c%c%c%c", static_cast<char>(m_title_id >> 24), static_cast<char>(m_title_id >> 16),
+        static_cast<char>(m_title_id >> 8), static_cast<char>(m_title_id));
```

Comment:
TMDReader::GetGameID does something similar, but it also appends the group ID, which is unwanted.

Tbh, I don't understand why we're doing that, when literally every other software that interacts with the Wii, homebrew or not, ignores the group ID.